style-dictionary/color.config.js

```diff
@@ -0,0 +1,32 @@
+const StyleDictionary = require("style-dictionary");
+const { template } = require("lodash");
+const fs = require('fs');
+const getJSON = require("./utils/getJSON");
+const colorTokens = getJSON("./packages/uswds-tokens/colors");
+
+const USWDS_COLORS = StyleDictionary.extend({
+  source: ["packages/uswds-tokens/colors/*.json"],
+  platforms: {
+    "scss/colors": {
+      transformGroup: "scss",
+      buildPath: "packages/uswds-core/src/styles/tokens/color/",
+      files: colorTokens.map((colorToken) => ({
+        destination: `_${colorToken}.scss`,
+        mapName: `system-color-${colorToken}`,
+        format: "custom/format/color-map",
+        filter: {
+          attributes: {
+            category: colorToken
+          }
+        }
+      })),
+    },
+  },
+})
+
+USWDS_COLORS.registerFormat({
+  name: 'custom/format/color-map',
+  formatter: template(fs.readFileSync(`${__dirname  }/templates/scss-color-map.template`))
+});
```

Comment:
## Custom Formater: Alternate solution

Currently, our sass color maps use unquoted keys for number values. Sass [recommends](https://arc.net/l/quote/prbozafo) we use quoted values for consistency and to avoid confusion for string values.

In order to output unquoted values, I needed to use this custom formatted with a custom template.  This also introduced a `lodash` dependency.

We can alternatively update the `get-system-color()` sass function with a one-line change that'll allow it to accept the color token keys as quoted values.

```diff
// get-system-color.scss

  $color-family: general.smart-quote($color-family);
+ $color-grade: #{$color-grade};
  $color-variant: general.smart-quote($color-variant);
```
---

Alternative solution outline:
- Remove custom formatted and template in favor of style dictionary `scss/map-deep` formatter.
- Remove `lodash` dependency.
- Add one line change to `get-system-color.scss`.
- No perceived visual regressions before or after updating color tokens.

Reply:
This approach was presented in [Dev Sync](https://docs.google.com/document/d/172TU5BVS9kGZQDuPcAhGoZjaTd2ngzcTayq4WRRTBUs/edit#heading=h.lkgdjynzaimo) (l🔒) and approved.